### DISCUSSION TITLE
Is there a list/repository of allowed custom fonts?

### DISCUSSION THREAD

**Discussion by luckymachi** [tectonic-typesetting/tectonic#1071] [Q&A]
As the question from the title implies, I was wondering if there was an official list that contains which fonts can be used with Tectonic and how to implement them. I am using the Nerd Font variant of JetBrains Mono and tried to use it, but had no luck with it. Any help to see if it can be installed or not as well as which other fonts can work would be greatly appreciated!

Edit: all fixed after checking out some commands and getting some help regarding how fontspec works!

**Comment by ararunaufc** [comment]
Great, now I will never know how... 😞
